resources/source-files.ditamap

```diff
@@ -138,6 +138,7 @@
   <keydef keys="plugin-anttarget" href="../topics/plugin-anttarget.dita"/>
   <keydef keys="plugin-benefits" href="../topics/plugin-benefits.dita"/>
   <keydef keys="plugin-coding-conventions" href="../topics/plugin-coding-conventions.dita"/>
+  <keydef keys="plugin-common" href="../topics/plugin-common.dita"/>
```

Comment:
Not sure we need a new file for this.

We have an existing warehouse topic for re-usable task components: [conref-task.dita](https://github.com/dita-ot/docs/blob/develop/resources/conref-task.dita).

Reply:
Changed to use conref-task.dita instead.